pkgs/test_process/pubspec.yaml

```
@@ -1,8 +1,9 @@
 name: test_process
 version: 2.1.1
 description: |
```

```suggestion
description: >-
```

Remove all newlines. We should do this everywhere.